src/utilities/mapUtilities.js

```diff
@@ -10,24 +10,28 @@ const generateConnectionData = (connection, tileId) => {
     return {
       osmStopId: connection[0].id.toString(),
       gtfsStopId: connection[1].id.toString(),
-      tileId: tileId
+      tileId: tileId,
     };
   } else if (connection[0].isOsm === false && connection[1].isOsm === true) {
```

Comment:
} else if (!(connection[0].isOsm)  && connection[1].isOsm ) {
?

Reply:
To się poprawi przy okazji refaktoringu.